runtime/server/x86/frontend/fft.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016 HR
+
+#ifndef FRONTEND_FFT_H_
+#define FRONTEND_FFT_H_
+
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+
+namespace wenet {
+
+// Fast Fourier Transform
+
+// #define M_PI 3.141592653589793238462643383279502
+
```

Comment:
Missing documentation.

Reply:
will use new fft later, so just leave it here.